### ISSUE TITLE
When changing Discord Username, all channels get renamed to own Username

### ISSUE THREAD

**Issue by ptgms** [mautrix/discord#9]
My discord nitro has run out recently and it showed me a rather weird issue. All the channels with DMs have renamed themselves to my own username rather than the username of the people I am texting. This issue presented itself like this:

1) Change tag (like example#1234)
2) Send a message to anyone on Discord
3) All chats will update to say "example#1234 (D)"

Maybe it would be cool to add like a "resync" command to circumvent this

**Comment by tulir** [comment]
Is the bridge up to date?

**Comment by ptgms** [comment]
Yes, it is fully up to date. I pulled the latest docker images just before opening this issue and testing it one more time.

**Comment by tulir** [comment]
Changing the username doesn't seem to change any room names for me, are you sure the bridge is up to date? (check with the `version` command in the management room)

**Comment by ptgms** [comment]
I was incredibly stupid I am sorry D:
I was indeed running an old version - version 0.0.1. The docker Image changed from the original mautrix-discord and I didn't update. Sorry again works now.
